selection.go

```diff
@@ -298,3 +298,16 @@ func (s *controlledSelector) HandleBindingRequest(m *stun.Message, local, remote
 		s.PingCandidate(local, remote)
 	}
 }
+
+type liteSelector struct {
+	pairCandidateSelector
+}
+
+// A lite selector should not contact candidates
+func (s *liteSelector) ContactCandidates() {
+	if _, ok := s.pairCandidateSelector.(*controllingSelector); ok {
+		// TODO: implement lite controlling agent. For now falling back to full agent.
```

Comment:
Can you create a github issue and tag it in the code base and just make the comment `pion/ice#$ISSUE_ID` otherwise we will never come back to it :)

I added `godox` to `pion/webrtc` but not all the dependencies yet!